CONTRIBUTING.md

```diff
@@ -228,4 +229,8 @@ Please run your code through:
 7. [Open a Pull Request](https://help.github.com/articles/using-pull-requests/)
     with a clear title and description against the `master` branch. All tests must be passing before we will review the PR.
 
+8. ## Code Reviews<a name="code-reviews"></a>
+If you can, please look at open PRs and review them. Give feedback and help us merge these PRs much faster! If you don't know how, Github has some <a href="https://help.github.com/articles/about-pull-request-reviews/">great information on how to review a Pull Request.</a>
```

Comment:
For the external link, Markdown syntax (`[text](link)`) might fit better with the rest of the document, although HTML technically works there.  In this doc regular HTML *is* used for making standalone anchors like
```HTML
<a name="roadmap"></a
```
because I believe there's no Markdown way to do it
```md
(#roadmap)  <-- doesn't work
```

Reply:
i see.. so should i update the file with markdown? i see that this same way is being used on line 7 so i figured it was ok to use. i appreciate the feedback tho 😄  let me know what i need to do to update this

Reply:
Yeah, I would use markdown like:
```
... great information on [how to review a Pull Request](https://help.github.com/articles/about-pull-request-reviews/).
```
This PR might not be merged though because there was one submitted just before it that covers the same thing.